app/components/obs/inputs/GenericFormTabbed.vue.ts

```diff
@@ -0,0 +1,59 @@
+import Vue from 'vue';
+import { Component, Prop } from 'vue-property-decorator';
+import { ISettingsSubCategory } from 'services/settings';
+import Tabs, { ITab } from 'components/Tabs.vue';
+import GenericForm from './GenericForm.vue';
+
+@Component({ components: { GenericForm, Tabs } })
+export default class GenericFormTabbed extends Vue {
+  @Prop() value: ISettingsSubCategory[];
+  currentTab: string = 'Streaming';
+
+  get tabs() {
+    const tabs = this.value
+      .filter(val => val.nameSubCategory !== 'Untitled')
+      // Exclude audio tabs since they're multiple of these and we only want one
+      .filter(val => !val.nameSubCategory.startsWith('Audio - Track'))
+      .map(toTab);
+
+    /*
+     * Insert "Audio" tab as the 3rd tab, since it looks more intuitive this way.
+     * TODO: I'd avoid mutation, but since we're not sold on that idea yet, we rather do it like
+     *  this instead of adding complexity or introducing deps.
+     */
+
+    const audioTab: ITab = { name: 'Audio', value: 'Audio' };
```

Comment:
I have a concern about you name this component as `GeneriggFormTabbed` but with this `Audio` tab handler and `currentTab: string = 'Streaming';` it looks like a specific component for streaming settings.

I see 2 options here:
1) Rename component to `StreamingSettingsForm`
2) Leave it as `GenericFormTabbed` but and make it reusable. (Move all special handlers for audio and default tab in SettingsService)

Reply:
Went with #2.